code/modules/ninja/energy_katana.dm

```diff
@@ -68,15 +68,14 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
-	block_chance = 50
 	slot_flags = SLOT_BELT
 	sharpness = IS_SHARP
 	max_integrity = 200
 	resistance_flags = LAVA_PROOF | FIRE_PROOF | ACID_PROOF
 	bypass_density = TRUE
 	var/datum/effect_system/spark_spread/spark_system
 
-/obj/item/dash/energy_katana/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+/obj/item/dash/energy_katana/afterattack(atom/target, mob/user, proximity_flag, click_parameters) //there should be a cooldown for this, it can be spammed for LOTS OF NOISE and LOTS OF SPARKS
```

Comment:
Either add a cooldown or make an issue report, don't add an unnecessary comment here in the code where it'll never be read again.

Reply:
woh